demos/text_detection_demo/src/text_detection.cpp

```diff
@@ -183,57 +211,82 @@ cv::Mat decodeImageByJoin(const std::vector<float> &cls_data, const std::vector<
 }
 }  // namespace
 
-std::vector<cv::RotatedRect> postProcess(const InferenceEngine::BlobMap &blobs, const cv::Size& image_size,
+std::vector<cv::RotatedRect> postProcess(const InferenceEngine::BlobMap &blobs,
+                                         const cv::Size &image_size, const cv::Size &input_shape,
                                          float cls_conf_threshold, float link_conf_threshold) {
     const int kMinArea = 300;
     const int kMinHeight = 10;
 
     std::string kLocOutputName;
     std::string kClsOutputName;
 
+    std::vector<cv::RotatedRect> rects;
```

Comment:
I don't think this variable is needed. Just `return` the appropriate value at the end of each branch.